### PR TITLE
Processor 2 almost done, negative bit not sure if working fine.

### PR DESCRIPTION
**Pull request by mhdelta**

### CODE REVIEW

**Comment by mhdelta** [comment]
solves #3

File: Processor/Entrega2/ALU.vhd [hunk 1]
```diff
@@ -89,7 +89,7 @@ process(ALUOP,Oper1,Oper2)
 			--SAVE 57
 			when "111001" => Salida <= Oper1 + Oper2;
 			--RESTORE 58
-			when "111010" => Salida <= Oper1 + Oper2;
+			when "111101" => Salida <= Oper1 + Oper2;
```
Comment (h3ct0rjs) [inline Processor/Entrega2/ALU.vhd]: Why Did you change this line it  should  be 111010 :thinking:

Comment (h3ct0rjs) [reply Processor/Entrega2/ALU.vhd]: @mhdelta You were right :smile:
